AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3389,6 +3389,7 @@ bmwfaq.org#?#.Notices[-ext-has='a[href="//www.bmwfaq.org/help/cookies"]']
 ! JS rules
 !
 group.rwe#%#document.cookie = "gdpr-settings-cookie=1; path=/;";
+mein-etikett.warsteiner.de#%#AG_onLoad(function() { confirmAge = confirmSocial } );
```

Comment:
better wrap to condition `if` to avoid an error after age confirmation
![image](https://user-images.githubusercontent.com/8361299/74174431-9a60b800-4c3c-11ea-96d5-fab9bad9bf5a.png)